### PR TITLE
Indeed Job Scraping and removing duplicate headers after parsing

### PR DESCRIPTION
**Pull request by jomanozgml**

### CODE REVIEW

File: indeed_fetch.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return title
+
+# async def get_header(page):
+#     header = await page.querySelector('div.jobsearch-JobInfoHeader-title-container')
+#     headerText = await page.evaluate('(element) => element.innerText', header)
+#     print('Position: ', headerText)
+#     return headerText
+
+async def get_description(page):
+    description = await page.querySelector('div.jobsearch-JobComponent-description')
```
Comment (kovarny) [inline indeed_fetch.py:16]: suggest we use tag names instead of classes to make it more generic

File: fetch_vacancy.py [hunk 2]
```diff
@@ -18,28 +20,28 @@ def fetch_vacancy(data, context):
...
+        title, text = asyncio.get_event_loop().run_until_complete(fetch_vacancy_details())
+    
+    else:
+        soup = get_page(vacancy_url)
+        title = get_og_title(soup)
+        desc = get_og_description(soup)
+        tags = soup.select("p, h2, h3, h4, h5, h6, ul")
+        # print(tags)
+        # [type(item) for item in list(soup.children)]
+        text = [title+'\n'] + [item.text+'\n' for item in list(tags)]
```
Comment (kovarny) [inline fetch_vacancy.py:42]: let's split this into separate function in a separate file

File: fetch_vacancy.py [hunk 3]
```diff
@@ -18,28 +20,28 @@ def fetch_vacancy(data, context):
...
-
-    # [type(item) for item in list(soup.children)]
-    text = [item.text+'\n' for item in list(tags)]
-
+    if 'indeed' in vacancy_url:
+        async def fetch_vacancy_details():
+            page = await get_page_indeed(vacancy_url)
+            return await get_title(page), await get_description(page)
+            
+        title, text = asyncio.get_event_loop().run_until_complete(fetch_vacancy_details())
```
Comment (kovarny) [inline fetch_vacancy.py:33]: can this be one separate function too?

**Review by kovarny — CHANGES_REQUESTED** [review]

**Comment by jomanozgml** [comment]
# Description: Scrapes a URL and returns the HTML source from dynamic page using Scrape.do
# Free Plan comes with With 5 concurrent requests, a total of 1000 requests per month
# As application user scales up alternative required: Pyppeteer or Selenium or Node.js methods

File: page_parser.py [hunk 4]
```diff
@@ -1,54 +1,19 @@
...
 def get_page(url):
-    """Scrapes a URL and returns the HTML source.
-    
-    Args:
-        url (string): Fully qualified URL of a page.
-    
-    Returns:
-        soup (string): HTML source of scraped page.
-    """
-
```
Comment (kovarny) [inline page_parser.py:14]: I suggest we still have the func definition comments

**Review by kovarny — APPROVED** [review]

**Pull request merged**